### ISSUE TITLE
Left navigation SC links for 2.4.7 through 2.4.10 are incorrect

### ISSUE THREAD

**Issue by lamador** [w3c/wcag#1043]
The left navigation pane has the following link issues:

- SC 2.4.7 is labeled as and links to "Location" page instead of "Focus Visible".
- SC 2.4.8 is labeled as and links to "Link Purpose (Link Only)" page instead of "Location".
- SC 2.4.9 is labeled as and links to "Section Headings" page instead of "Link Purpose (Link Only)".
- SC 2.4.10 is labeled as and links to "Focus Visible (Enhanced)" page instead of "Section Headings".

**Comment by alastc** [comment]
I assume you mean that this location:
https://w3c.github.io/wcag/guidelines/22/#focus-visible 

I can't replicate this, it works for me. Unless there's some more information to help work out the issue (e.g. URL, browser /AT), I'll close for now.

**Comment by patrickhlauke** [comment]
This is related to the interplay between WCAG 2.1 and the Understanding WCAG 2.1 https://www.w3.org/WAI/WCAG21/Understanding/

It seems that somehow, Focus Visible (Enhanced) (which is NOT part of 2.1, but upcoming 2.2) is already now visible/published under the WCAG 2.1 Understanding doc jttps://www.w3.org/WAI/WCAG21/Understanding/focus-visible-enhanced and it is numbered there as 2.4.10.

However, 2.4.10 is already a 2.1. SC, Section Headings. It now gets very confusing if you start off on the WCAG 2.1 spec at, say, 2.4.10 https://www.w3.org/TR/WCAG21/#section-headings then follow the "Understanding Section Headings" - you land on https://www.w3.org/WAI/WCAG21/Understanding/section-headings.html which now says this is *2.4.9* Section Headings ... and well, it all gets borked a bit from there.

comparing the Understanding WCAG 2.1 index for section 2.4 and the actual WCAG 2.1 spec, it seems that in the former what was actually 2.4.7 Focus Visible has disappeared, subsequent SC numbers have been shifted left, and the new Focus Visible (Enhanced) published prematurely as the new 2.4.10

This is not obvious if you usually just primarily navigate the WCAG 2.1 spec and just jump in/out of the relevant understanding doc. But if you navigate the 2.1 Understanding doc itself, you fall into this weird parallel universe.

![Comparison of WCAG 2.1 index for 2.4 on the left and the index for 2.4 in Understanding WCAG 2.1 on the right showing the discrepancy](https://user-images.githubusercontent.com/895831/75082658-92dac200-550c-11ea-97cf-59c4651e22fe.png)

**Comment by patrickhlauke** [comment]
the fix here is essentially to remove the premature Focus Visible (Enhanced) from the Understanding WCAG 2.1 (and also, in that https://www.w3.org/WAI/WCAG21/Understanding/focus-visible-enhanced at the moment, many links are borked as well), ensure that the current 2.4.7 Focus Visible is listed properly in the Understanding document, and that the numbers for the following SCs are as they should be and not bumped one digit to the left

**Comment by alastc** [comment]
Ah, thanks, without the starting point that was confusing!

**Comment by JAWS-test** [comment]
Further problems arise in this context:
* https://www.w3.org/WAI/WCAG21/Understanding/timing-adjustable: link "Previos SC: Content on Hover or Focus" jump to https://www.w3.org/WAI/WCAG21/Understanding/keyboard
* https://www.w3.org/WAI/WCAG21/Understanding/use-of-color: link "Previos SC" jump to https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

and so on

**Comment by alastc** [comment]
I think the big problem is that the current 2.4.7 has been updated, which has thrown off the understanding navigation. A lot of it is automated by the build-process, so once that's fixed it should be resolved. @michael-n-cooper is aware, and back in the office this week.

**Comment by michael-n-cooper** [comment]
This was a generator error, fixed by 34f2734136dc4beb2efe5cbef821447469ecc871

**Comment by ziachowdury** [comment]
> The left navigation pane has the following link issues:
> 
> * SC 2.4.7 is labeled as and links to "Location" page instead of "Focus Visible".
> * SC 2.4.8 is labeled as and links to "Link Purpose (Link Only)" page instead of "Location".
> * SC 2.4.9 is labeled as and links to "Section Headings" page instead of "Link Purpose (Link Only)".
> * SC 2.4.10 is labeled as and links to "Focus Visible (Enhanced)" page instead of "Section Headings".
